src/modules/maintenance/ConfigModule/ConfigContainers/ConfigEditor.js

```diff
@@ -662,7 +662,7 @@ function ConfigSection(props) {
 					</div>
 				</Col>
 			</Row>
-			<FormGroup tag="fieldset" disabled={props.isSubmitting}>
```

Comment:
@petrKavulok Why? I thought that `<FormGroup>` is a reactstrap tag, or were there any clashes with bootstrap5?

Reply:
Hi, it `<FormGroup>` is using Bootstraps class names and respective styling. Form specific layout classes were dropped in B5. 
<img width="826" alt="Screenshot 2023-08-09 at 9 57 48" src="https://github.com/TeskaLabs/asab-webui/assets/79644454/92a4b590-789a-4152-b283-29866f5bcb92">

https://getbootstrap.com/docs/5.1/migration/#forms

Reply:
Yes, but we are using reactstrap component here. And it is still present in reactstrap v9, which should be fully compatible with bootstrap v5